From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 16 Nov 2017 20:46:00 +0000
Subject: ALSA: timer: Avoid ABI change in 4.13.13
Forwarded: not-needed

Commit 9b7d869ee5a7 ("ALSA: timer: Limit max instances per timer")
added two new members to the end of struct snd_timer, and this changes
ABI for many exported symbols.  However, snd_timer is always allocated
in the snd-timer module, which is the same place the new members are
used.  We can safely hide the change from genksyms.

---
--- a/include/sound/timer.h
+++ b/include/sound/timer.h
@@ -90,8 +90,10 @@ struct snd_timer {
 	struct list_head ack_list_head;
 	struct list_head sack_list_head; /* slow ack list head */
 	struct tasklet_struct task_queue;
+#ifndef __GENKSYMS__
 	int max_instances;	/* upper limit of timer instances */
 	int num_instances;	/* current number of timer instances */
+#endif
 };
 
 struct snd_timer_instance {
